### ISSUE TITLE
Prepatch error

### ISSUE THREAD

**Issue by fisken1991** [2072/LibNameplateRegistry-1.0#1]
Getting this error when mouseover something
```
`5x ...ibNameplateRegistry-1.0/LibNameplateRegistry-1.0-20.lua:620: attempt to call upvalue 'GetMouseFocus' (a nil value)
[string "@TargetNameplateIndicator/Libs/LibNameplateRegistry-1.0-20/LibNameplateRegistry-1.0/LibNameplateRegistry-1.0.lua"]:620: in function `?'
[string "@TargetNameplateIndicator/Libs/LibNameplateRegistry-1.0-20/LibNameplateRegistry-1.0/LibNameplateRegistry-1.0.lua"]:892: in function <...ibNameplateRegistry-1.0/LibNameplateRegistry-1.0.lua:891>

Locals:
self = <table> {
 callbacks = <table> {
 }
 UsedCallBacks = 3
 EventFrame = Frame {
 }
 mixinTargets = <table> {
 }
}
unitName = ""
mouseoverNameplate = nil
data = nil
(*temporary) = nil
(*temporary) = "attempt to call upvalue 'GetMouseFocus' (a nil value)"
GetMouseFocus = nil
WorldFrame = WorldFrame {
}
ActivePlates_per_frame = <table> {
  = <table> {
 }
  = <table> {
 }
  = <table> {
 }
  = <table> {
 }
  = <table> {
 }
  = <table> {
 }
  = <table> {
 }
  = <table> {
 }
  = <table> {
 }
  = <table> {
 }
  = <table> {
 }
  = <table> {
 }
  = <table> {
 }
  = <table> {
 }
  = <table> {
 }
  = <table> {
 }
  = <table> {
 }
  = <table> {
 }
  = <table> {
 }
  = <table> {
 }
  = <table> {
 }
  = <table> {
 }
  = <table> {
 }
  = <table> {
 }
  = <table> {
 }
  = <table> {
 }
  = <table> {
 }
}
HighlightFailsReported = false
INFO2 = 4
WARNING = 2
`
```

**Comment by 2072** [comment]
This is fixed in 1.0.21 which is not tagged yet, you can get the alpha
package on wowace:
https://www.wowace.com/projects/libnameplateregistry-1-0/files

**Comment by 2072** [comment]
Closing: 1.0.21 was released yesterday.
